build.gradle

```diff
@@ -140,7 +140,10 @@ subprojects {
     }
 }
 
-def feign_version = '11.5'
+def feign_version = '11.7'
```

Comment:
Perhaps we could do the upgrade in a separate PR? Not a big deal though.

Reply:
Sure

Reply:
Done